homeassistant/components/proxmoxve/strings.json

```diff
@@ -0,0 +1,28 @@
+{
+    "config": {
+      "step": {
+        "user": {
+          "title": "ProxmoxVE",
+          "description": "Please fill out the form and then press submit when you're ready",
```

Comment:
Not really a needed/critical description. If this was needed in general, we should resolve that in the UI to be something general instead (and not in an integration level solution).